samples/bash/src/main/resources/changelog.kts

```diff
@@ -27,5 +30,6 @@ changeLog {
                     export | grep " PWD"
                 """.trimIndent()
         }
+        bashAction(workingDir = "/tmp", file = File(Main::class.java.getResource("pwet.sh").toURI()))
```

Comment:
Note: `File(Main::class.java.getResource(...).toURI())` could fail

Reply:
yes, it therefore sets the changeSet to fail. This is the wanted behavior?